munet/mutest/userapi.py

```diff
@@ -122,7 +122,7 @@ def pause_test(desc=""):
 def act_on_result(success, args, desc=""):
     if args.pause:
         pause_test(desc)
-    elif success:
+    elif success or len(desc) == 0:
```

Comment:
Maybe I'm misreading this, but it seems like it's ignoring the currently executing user's request to go to the CLI when a failure occurs because the author of the failing test didn't provide a description. I'm not sure that makes sense to me.

Reply:
Please add a comment that indicates that lack of description means no logical error has occurred so we skip the last 2 checks.

Also please modify the docs for `desc` args in the API calls from:

```
desc (str) – description of test, if no description then no result is logged.
```
to:
```
desc (str) – description of test, if no description then step failure is not considered an error and no result is logged.
```